src/main/java/summaryPackage/RouteCalculation.java

```diff
@@ -0,0 +1,61 @@
+package summaryPackage;
+
+import calculation.CalculateCoordinates;
+import calculation.CalculateP2PDistance;
+import cli.CliProperties;
+import http.Client;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import model.LocationPoint;
+import model.Route;
+import reader.DataReader;
+import reader.RouteSerializer;
+import utils.ResponseUtils;
+import utils.Utilities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static utils.ResponseUtils.validateJsonAgainstSchema;
+
+@Slf4j
+public class RouteCalculation {
+
+  public List<Route> newPoints(CliProperties cliProperties) {
+    LocationPoint locationPoint;
+    LocationPoint locationPoint1;
+    Client client = new Client();
+    DataReader dataReader = new DataReader(); // read data base from file
+    RouteSerializer routeSerializer = new RouteSerializer();
+    List<String> points =
+        dataReader.readFormattedJsonFile(
+            (cliProperties.getNumberOfRoutes()) * 2, cliProperties.getInputFile());
+    List<Route> routes = new ArrayList<>();
+    double step = Utilities.calculateStep(cliProperties.getSpeed(), cliProperties.getInterval());
+    int id = 0;
+    for (int i = 0; i < (cliProperties.getNumberOfRoutes()) * 2; i = i + 2) {
```

Comment:
I think you can calculate it before the for loop  (cliProperties.getNumberOfRoutes()) * 2; and assign it to some local variable and then use in for loop this variable